problem2/problem2_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "./problem2"
+describe Problem2 do
+	it "Largest 2 Numbers" do
+		example1 = Problem2.new(11,2,3,4,12,3,14)
```

Comment:
All the problem state `Given an array`. This signature does not accept an array. In real life we would not do variable arguments length to pass in arrays since we don't actually know the number of elements in the array. Using the `splat operator` will only introduce computational / maintainability overhead in this case.